styles.css

```diff
@@ -281,19 +304,19 @@ li {
   width: 89px;
   height: 48px;
   background-color: #7d73fc;
-  font-family: 'Roboto', sans-serif;
   font-weight: 500;
   font-size: 12px;
   line-height: 24px;
 }
 
 button:hover {
-  background-color: #979123;
-  border: 2px solid #fcf239;
+  border: 2px solid #2415f8;
+  background: #7d73fc;
+  box-shadow: 0 8px 16px rgba(125, 115, 252, 0.48);
 }
```

Comment:
It would be nice if you for all of your buttons add pointer cursor to improve the UX, you can use the next attributes to add it:
```
button:hover {
  cursor: pointer;
}
```

- [ ] [OPTIONAL] Add pointer cursor when you hover buttons.

Reply:
Well solved